classes/Collection.js

```diff
@@ -57,6 +73,22 @@ class Collection {
 
       await config.update(newContent, sha)
 
+      // Delete collection in nav if it exists
+      const nav = new File(this.accessToken, this.siteName)
+      const dataType = new DataType()
+      nav.setFileType(dataType)
+      const { content:navContent, sha:navSha } = await nav.read(NAV_FILE_NAME)
+      const navContentObject = yaml.safeLoad(base64.decode(navContent))
+
+      for (let i = 0; i < navContentObject.links.length; i++) {
+        if (navContentObject.links[i].collection === collectionName) {
+          navContentObject.links.splice(i,1)
+          const newNavContent = base64.encode(yaml.safeDump(navContentObject))
+          await nav.update(NAV_FILE_NAME, newNavContent, navSha)
+          break
+        }
+      }
```

Comment:
Nit: generally it's safer to practice immutable programming (for a more detailed reason why: https://stackoverflow.com/questions/34385243/why-is-immutability-so-important-or-needed-in-javascript) so what we could do here, instead of unnaturally breaking a for loop, is:

```
const newNavLinks = navContentObject.links.filter(< filter out collection name>)
const newNavContentObject = {
  ...navContentObject,
  links: newNavLinks,
}
```

Since the code here works and is performant I don't have a problem with it so I'll leave it up to you

Reply:
Oh yeah this way makes more sense. I've changed this in both resourceroom and collections in https://github.com/isomerpages/isomercms-backend/pull/80/commits/fb58d147288767db47d61d013dc4d15afd96c57d